src/config.rs

```diff
@@ -696,15 +696,15 @@ impl Default for MetricConfig {
 }
 
 #[derive(Serialize, Deserialize)]
-#[serde(remote = "LogLevelFilter")]
+#[serde(remote = "Level")]
 #[serde(rename_all = "kebab-case")]
 pub enum LogLevel {
```

Comment:
**This is a breaking change.**

`slog` does not have the concept of an `Off` setting, and changes the name of `Warn`, while introducing a `Critical` level.

Reply:
`Off` is barely used, I think it's OK to just removing it. But `Warn` is commonly used, better make it compatible.

Reply:
:+1:

Reply:
can we do a conversion internally?  some users now use `warn` in production.